src/app/(BottomTabNavigation)/AllCases/EligibilityForm/[caseUid].tsx

```diff
@@ -19,39 +35,67 @@ export default function EligibilityForm() {
       });
     }
   };
+
   return (
-    <View style={styles.container}>
+    <ScrollView style={styles.container}>
+      {caseData && (
+        <>
+          <Image style={styles.image} source={{ uri: caseData.imageUrl }} />
+          <Text style={styles.title}>{caseData.title}</Text>
+        </>
+      )}
+
       <View style={styles.textContainer}>
```

Comment:
Create a container with an ~84% width, so all items can span a proportionally equal width, regardless of screen size. That way we don't have to center items using pixel margins etc.